src/plugins/intel_cpu/tests/functional/single_layer_tests/classes/eltwise.cpp

```diff
@@ -139,34 +158,58 @@ void EltwiseLayerCPUTest::SetUp() {
+                shape = pShape.get_shape();
+            } else {
+                ASSERT_TRUE(pShape.rank().is_static());
+                shape = std::vector<size_t>(pShape.rank().get_length(), 1);
+                for (size_t i = 0; i < pShape.size(); ++i) {
+                    if (pShape[i].is_static()) {
+                        shape[i] = pShape[i].get_length();
+                    }
                 }
             }
+
+            auto data_tensor = generate_eltwise_input(netType, shape);
+            if ((netType == ElementType::i8) || (netType == ElementType::u8)) {
+                auto data_ptr = reinterpret_cast<uint8_t*>(data_tensor.data());
+                std::vector<uint8_t> data(data_ptr, data_ptr + ngraph::shape_size(shape));
+                secondaryInput = ngraph::builder::makeConstant(netType, shape, data);
+            } else if ((netType == ElementType::i16) || (netType == ElementType::u16)) {
+                auto data_ptr = reinterpret_cast<uint16_t*>(data_tensor.data());
+                std::vector<uint16_t> data(data_ptr, data_ptr + ngraph::shape_size(shape));
+                secondaryInput = ngraph::builder::makeConstant(netType, shape, data);
+            } else if ((netType == ElementType::i32) || (netType == ElementType::u32)) {
+                auto data_ptr = reinterpret_cast<uint32_t*>(data_tensor.data());
+                std::vector<uint32_t> data(data_ptr, data_ptr + ngraph::shape_size(shape));
+                secondaryInput = ngraph::builder::makeConstant(netType, shape, data);
+            } else if (netType == ElementType::f16) {
+                auto data_ptr = reinterpret_cast<ov::float16*>(data_tensor.data());
+                std::vector<ov::float16> data(data_ptr, data_ptr + ngraph::shape_size(shape));
+                secondaryInput = ngraph::builder::makeConstant(netType, shape, data);
+            } else {
+                auto data_ptr = reinterpret_cast<float*>(data_tensor.data());
+                std::vector<float> data(data_ptr, data_ptr + ngraph::shape_size(shape));
+                secondaryInput = ngraph::builder::makeConstant(netType, shape, data);
+            }
+            break;
         }
-        if (netType == ElementType::i32) {
-            auto data_tensor = generate_eltwise_input(ElementType::i32, shape);
-            auto data_ptr = reinterpret_cast<int32_t*>(data_tensor.data());
-            std::vector<int32_t> data(data_ptr, data_ptr + ngraph::shape_size(shape));
-            secondaryInput = ngraph::builder::makeConstant(netType, shape, data);
-        } else {
-            auto data_tensor = generate_eltwise_input(ElementType::f32, shape);
-            auto data_ptr = reinterpret_cast<float*>(data_tensor.data());
-            std::vector<float> data(data_ptr, data_ptr + ngraph::shape_size(shape));
-            secondaryInput = ngraph::builder::makeConstant(netType, shape, data);
+        case ngraph::helpers::InputLayerType::NONE: {
```

Comment:
I would prefer to avoid `ngraph::helpers::InputLayerType::NONE` introduction, given this is common enum: now each test which uses it as a parameter should care about how `NONE` value should be handled. 
For our needs it is just enough to add condition on `BITWISE_NOT` inside `EltwiseLayerCPUTest::SetUp`
It will also allow to avoid test instances duplication for `BITWISE_NOT` operation

Reply:
done, thanks